py3status/modules/whatismyip.py

```diff
@@ -13,14 +14,13 @@
         (default 'ip')
     negative_cache_timeout: how often to check again when offline (default 2)
     timeout: how long before deciding we're offline (default 5)
-    url: change IP check url (must output a plain text IP address)
-        (default 'http://ultrabug.fr/py3status/whatismyip')
```

Comment:
We could... but why having two ways to detect IP address when we could just have one? 🙂 
It is a wonderful and simple service, no doubt, but this algorithm becomes quite complicated if we want to support both URLs :)